src/declarations/rewards/rewards.did

```diff
@@ -264,6 +264,10 @@ type UsageAwardEvent = record {
 };
 type UsageAwardState = record {
   snapshots : vec UserSnapshot;
+    /// Pseudonym for user who introduced the caller to Oisy.
+  referred_by : opt nat32;
+    /// User's code for inviting others to the platform, and referral stats. 
+  referrer_info : opt ReferrerInfo;
```

Comment:
Note: These types are not exposed to users and don't affect the front-end integration.